app/graphql/albums/type_defs.js

```diff
@@ -2,7 +2,7 @@ const { gql } = require('apollo-server');
 
 const rootTypes = gql`
   extend type Query {
-    album(id: Int): Album!
+    album(id: Int): [Album]!
```

Comment:
mmm this is a little bit strange, why?

Reply:
Because I made a mistake, but I already corrected it, I'm sorry!